src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -72,7 +72,7 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
                         data-testid="delete-annotation-activity"
                         onClick={handleDeleteClick}
                     >
-                        <IconTrash color={bdlGray} />
+                        <Trash16 color={bdlGray} />
```

Comment:
@ChenCodes you will notice the new icons do not support `color` in this way, but its likely the color is correct because of the defaults set by design. this should be cleaned up. If you need a custom color here (or to handle `:hover`) use SASS

Reply:
Thanks for the heads up, I'll put a fix in for this.